Add support policy (#805)
FreeApophis/Funcky: SupportPolicy.md

# Support Policy

Funcky is a stable base-library which tries to support many scenario's in the .NET world.

## What Frameworks do we support?

We support the all the activly supported modern Framworks, .NET Standard 2.1, 2.0 And.NET 4.8+

> * We only remove support for tagrgetframeworks on major version changes, to avoid breaking changes for the consumers.

Reviewer: ```suggestion
* We only remove support for target frameworks on major version changes, to avoid breaking changes for the consumers.
```

## Testing

Testing with old framworks which are not supported anymore is getting painful, we therefore only test with frameworks which are still in active support.

## Next removal

> Funcky 4 will remove the following Targetframeworks.

Reviewer: ```suggestion
Funcky 4 will remove the following target frameworks:
```

* net7.0
* net6.0
* net5.0

> * netcoreapp3.1;

Reviewer: ```suggestion
* netcoreapp3.1
```
